indexing/metadata.md

```diff
@@ -18,44 +18,199 @@ Including the following tags and metadata in each of your pages will improve the
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#content-type">Content Type</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#searchgov-custom-fields">Search.gov Custom Fields</a> 
+        </li>
+      </ul>
+    </div>
+  </div>
+</div>
+
+### Fields by Type
+
+|[Standard Fields](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta/name)|[OpenGraph Fields](https://ogp.me/)|[DublinCore Fields](https://www.dublincore.org/)|
+|:----|:----|:----|:----|
+|[`<main>`](#main-page-content)|[`og:title`](#title)|[`dcterms.keywords`](#tags--keywords)|
+|[`<title>`](#title)|[`og:description`](#description)|[`dc.subject`](#tags--keywords)|
+|[`description`](#description)|[`article:section`](#tags--keywords)|[`dcterms.subject`](#tags--keywords)|
+|[`keywords`](#tags--keywords)|[`article:tag`](#tags--keywords)|[`dcterms.audience`](#audience)|
+|[`robots`](#crawling-metadata)|[`article:published_time`](#created-date)|[`dc.type`](#content-type)|
+||[`article:modified_time`](#changed-date)|[`dcterms.type`](#content-type)|
+||[`og:image`](#thumbnail-image)|[`dc.date`](#created-date)|
+|||[`dc.date.created`](#created-date)|
+|||[`dcterms.created`](#created-date)|
+
+
+### Main Page Content
+We look for the following field:
+```
+<main>
+```
+- **Detail:** Allows the search engine to target the actual content of the page and avoid headers, sidebars and other page content not useful to search. [Read more about the <main> element here]({{ site.baseurl }}/indexing/how-search-engines-index-content-better-discoverability.html#main-element)
+- **Used In:** Query matching, term frequency scoring
+
+### Title
+
+We look for the following fields:
+```
+<title>
+<meta property="og:title" content="Example" />
+```
+- **Detail:** Unique title of the page. If you want to include the agency or section name, place that after the actual page title. We use the `og:title` property as the result title if it appears to be more substantive than the `<title>` tag. Note, Open Graph elements are used to display previews of your content in FaceBook and some other social media platforms.
+- **Data Gathering Logic:** We prefer whichever title field is longer between the two.
+- **Used In:** Query matching, term frequency scoring
+
+
+### Description
+
+We look for the following fields:
+```
+<meta name="description" content="Example" />
+<meta property="og:description" content="Example" />
+```
+- **Detail:** Your well crafted, plain language summary of the page content. This will often be used by search engines in place of a page snippet. Be sure to include the keywords you want the page to rank well for. Best to limit to 160 characters, so it will not be truncated. [Read more here](https://moz.com/learn/seo/meta-description). Note, Open Graph elements are used to display previews of your content in FaceBook and some other social media platforms.
```

Comment:
GSA style wants link text to be nouns. How about we change "[Read more here](link etc)." to "Read more technical [background and guidance](link goes here)."

Reply:
fixed